lib/pyfrc/mains/cli_deploy.py

```diff
@@ -21,111 +20,171 @@
 
 def relpath(path):
     '''Path helper, gives you a path relative to this file'''
-    return os.path.normpath(os.path.join(os.path.abspath(os.path.dirname(__file__)), path))
-    
+    return os.path.normpath(
+        os.path.join(os.path.abspath(os.path.dirname(__file__)), path))
+
+
 class PyFrcDeploy:
     """
         Uploads your robot code to the robot and executes it immediately
     """
-    
+
     def __init__(self, parser):
         ''' :type parser: argparse.ArgumentParser '''
-        parser.add_argument('--builtin', default=False, action='store_true',
-                                help="Use pyfrc's builtin tests if no tests are specified")
-        
-        parser.add_argument('--skip-tests', action='store_true', default=False,
-                            help="If specified, don't run tests before uploading code to robot (DANGEROUS)")
-        
-        parser.add_argument('--debug', action='store_true', default=False,
-                            help="If specified, runs the code in debug mode (which only currently enables verbose logging)")
-        
-        parser.add_argument('--nonstandard', action='store_true', default=False,
-                            help="When specified, allows you to deploy code in a file that isn't called robot.py")
-        
-        parser.add_argument('--nc', '--netconsole', action='store_true', default=False,
-                            help="Attach netconsole listener and show robot stdout (requires DS to be connected)")
-        
-        parser.add_argument('--nc-ds', '--netconsole-ds', action='store_true', default=False,
-                            help="Attach netconsole listener and show robot stdout (fakes a DS connection)")
-
-        parser.add_argument('--in-place', action='store_true', default=False,
-                            help="Overwrite currently deployed code, don't delete anything, and don't restart running robot code.")
-        
-        parser.add_argument('-n', '--no-version-check', action='store_true', default=False,
-                            help="If specified, don't verify that your local wpilib install matches the version on the robot (not recommended)")
-        
-        parser.add_argument('--robot', default=None,
-                            help="Set hostname or IP address of robot")
-        
-        parser.add_argument('--no-resolve', action='store_true', default=False,
-                            help="If specified, don't do a DNS lookup, allow ssh et al to do it instead")
-    
+        parser.add_argument(
+            '--builtin',
+            default=False,
+            action='store_true',
+            help="Use pyfrc's builtin tests if no tests are specified")
+
+        parser.add_argument(
+            '--skip-tests',
+            action='store_true',
+            default=False,
+            help=
+            "If specified, don't run tests before uploading code to robot (DANGEROUS)"
```

Comment:
What on actual earth happened here. I would go so far to consider this a yapf bug.